gcloud-java-datastore/src/test/java/com/google/gcloud/datastore/LocalGcdHelper.java

```diff
@@ -62,17 +64,85 @@
   public static final String DEFAULT_PROJECT_ID = "projectid1";
   public static final int PORT = 8080;
   private static final String GCD = "gcd-v1beta2-rev1-2.1.2b";
```

Comment:
Yea -- that was what I thought, and was wondering where you could safely re-use `GCD_VERSION` (or any of the others). 

Unless we have a good example, how about leaving as a string, and splitting apart when needed? It's a super simple refactor and means we have a good reason to do it...